libs/blocks/global-navigation/utilities/getUserEntitlements.js

```diff
@@ -125,30 +126,30 @@ const getSubscriptions = async ({ queryParams }) => {
 const getUserEntitlements = async ({ params, format } = {}) => {
   if (!window.adobeIMS?.isSignedInUser()) {
     entitlements = {};
-    return Promise.resolve(emptyEntitlements());
+    return Promise.resolve(emptyEntitlements(format));
   }
 
   const queryParams = getQueryParameters(params);
   if (entitlements[queryParams]) {
-    return format === 'raw'
+    return format === FORMAT_RAW
       ? entitlements[queryParams]
       : entitlements[queryParams]
         .then((res) => mapSubscriptionCodes(res))
-        .catch(() => emptyEntitlements());
+        .catch(() => emptyEntitlements(format));
   }
 
   entitlements[queryParams] = entitlements[queryParams]
    || new Promise((resolve) => {
-     getSubscriptions({ queryParams })
+     getSubscriptions({ queryParams }, format)
```

Comment:
```suggestion
     getSubscriptions({ queryParams, format })
```

Reply:
@mokimo  `getSubscriptions` expects two params, I didn't want to change its signature.